scripts/bootstrap-nixos.sh

```diff
@@ -322,8 +322,9 @@ if yes_or_no "Do you want to copy your full nix-config and nix-secrets to $targe
 	if yes_or_no "Do you want to rebuild immediately?"; then
 		green "Rebuilding nix-config on $target_hostname"
 		#FIXME there are still a gitlab fingerprint request happening during the rebuild
-		#$ssh_cmd -oForwardAgent=yes "cd nix-config && sudo nixos-rebuild --show-trace --flake .#$target_hostname" switch"
-		$ssh_cmd -oForwardAgent=yes "cd nix-config && just rebuild"
+		$ssh_cmd -oForwardAgent=yes "cd nix-config && sudo nixos-rebuild --show-trace --flake .#$target_hostname switch"
+		# FIXME: This fails because `just rebuild` tries to run `nix flake update nix-secrets` but the flake registry doesn't exist yet
```

Comment:
@DrymarchonShaun do you recall what the failure message is for this? This has been working okay for me so I'd like to test it against a vm at some point.

Reply:
> @DrymarchonShaun do you recall what the failure message is for this? This has been working okay for me so I'd like to test it against a vm at some point.

I don't remember exactly what the error is, its something about nix-secrets not existing in the registry. I think it has something to do with `just rebuild` trying to run `nix flake update nix-secrets`. It's only a problem in the initial minimal config, after the first build of the full config with `just rebuild` works fine.